packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
+  /// A pointer has contacted the screen and might begin to move.
   GestureDragDownCallback onDown;
+
+  /// A pointer has contacted the screen and has begun to move.
   GestureDragStartCallback onStart;
+
+  /// A pointer that is in contact with the screen and moving has moved again.
   _GesturePolymorphicUpdateCallback<T> onUpdate;
```

Comment:
Yeah, I poked around a bit at that.  It seems to work out by some sort of magic due to aligning typedefs.  I think we should make _GesturePolymorphicUpdateCallback a public type to simplify the story.  The whole class (_DragGestureRecognizer) is private, but I think these properties end up being public on a public class.  It's all a bit confusing.